pkgs/by-name/xo/xonsh/xontribs/xontrib-direnv/default.nix

```diff
@@ -0,0 +1,20 @@
+{
+  pkgs,
+  python3,
+}:
+python3.pkgs.buildPythonPackage {
+  pname = "xontrib-xonsh-direnv";
+  version = "1.6.1";
```

Comment:
1.6.1 is bugged. I've worked with 74th to create a 1.6.2 which I'm adding to my existing branch.

Reply:
removed from my branch